build/templates/_library.py.mako

```diff
@@ -51,7 +51,12 @@ class Library(object):
     def ${c_func_name}(${param_names_method}):  # noqa: N802
         with self._func_lock:
             if self.${c_func_name}_cfunc is None:
-                self.${c_func_name}_cfunc = self._library.${c_func_name}
+                try:
+                    self.${c_func_name}_cfunc = self._library.${c_func_name}
+                except AttributeError as e:
+                    raise AttributeError("A required function was not found in the instrument driver DLL. This might "
+                                         "be an indication that the version of the instrument driver is too old for "
+                                         "this version of the python API. Upgrade your instrument driver.") from e
```

Comment:
Please provide the preferred message text.

Reply:
I think you have enough context in the codegenerator to give a really specific error. Here's a suggestion of what that should look like based on the error string in `DriverNotInstalledError`:

> Function `niDCPower_InitializeWithChannels` was not found in the NI-DCPower runtime. Please visit http://www.ni.com/downloads/drivers/ to download a newer version and install it.' 

An additional question: do we want a `DriverTooOld` class for this or is `AttributeError` better?

Reply:
I don't have an opinion on the DriverTooOld class.
Do you know of an existing way to obtain the driver name (e.g. NI-DCPower) for this message?

Reply:
I added a DriverTooOldError class.